docs/data/base/components/autocomplete/AutocompleteIntroduction/css/index.js

```diff
@@ -278,7 +278,7 @@ function Styles() {
         width: 100%;
         border-radius: 12px;
         overflow: auto;
-        outline: 0px;
+        outline: 0;
```

Comment:
This is not detected by the stylelint parser, I created an issue for them: https://github.com/hudochenkov/postcss-styled-syntax/issues/31.

<img width="600" alt="SCR-20240528-rgpl" src="https://github.com/mui/material-ui/assets/3165635/9732883d-788b-45aa-a175-28096114d2aa">